lib/cli/commands/listtrades.ts

```diff
@@ -0,0 +1,45 @@
+import { Arguments } from 'yargs';
+import { callback, loadXudClient } from '../command';
+import { ListTradesRequest, ListTradesResponse, Trade } from '../../proto/xudrpc_pb';
+import Table, { HorizontalTable } from 'cli-table3';
+import colors from 'colors/safe';
+
+const HEADERS = [
+  colors.blue('rhash'),
+  colors.blue('trade quantity'),
+  colors.blue('trading pair'),
+  colors.blue('order type'),
+];
+
+const displayTrades = (trades: ListTradesResponse.AsObject) => {
+  const table = new Table({ head: HEADERS }) as HorizontalTable;
+  trades.tradesList.forEach((trade: Trade.AsObject) => {
+    const type = trade.makerOrder ? 'maker' : 'taker';
+    table.push([
+      trade.rHash,
+      trade.quantity,
```

Comment:
We should convert this from satoshis to whole units using the `satsToCoinsStr` utils method.